driver_scripts/driver_grid.hera.sh

```diff
@@ -115,6 +115,7 @@ if [ $gtype = uniform ]; then
   export add_lake=true         # Add lake frac and depth to orography data.
   export lake_cutoff=0.50      # return 0 if lake_frac <  lake_cutoff & add_lake=T
   export binary_lake=1         # return 1 if lake_frac >= lake_cutoff & add_lake=T
+  export ocn=${ocn:-"100"}     # use one of  "025", "050", "100", "500". Cannot be empty
```

Comment:
This line should be added to the scripts for the other machines.

Reply:
Those files are 'fixed' correct? If so, they should be placed somewhere in the official 'fixed' directories.

Reply:
When those ocean merge files are missing, the program fails, but the script continues. I added some error handling logic to two scripts so execution ceases. 

```
--- a/ush/fv3gfs_ocean_merge.sh
+++ b/ush/fv3gfs_ocean_merge.sh
@@ -15,12 +15,17 @@
     /
 EOF

+    echo Run ocean_merge program.

     time  ${exec_dir}/./ocean_merge
+    rc=$?

-    set -x
-    echo run orog 2nd time
-    set +x
+    if [[ $rc -ne 0 ]] ; then
+     echo FATAL ERROR running ocean_merge.
+     exit $rc
+    fi
+
+    echo Run orog 2nd time

     for tnum in '1' '2' '3' '4' '5' '6'
     do
```
 
and 

```
--- a/ush/fv3gfs_driver_grid.sh
+++ b/ush/fv3gfs_driver_grid.sh
@@ -226,12 +226,15 @@ if [ $gtype = uniform ] || [ $gtype = stretch ] || [ $gtype = nest ];  then
     fi
   fi

-
-       if [ $gtype = uniform ]; then
-               if declare -p ocn &>/dev/null;then
-                        $script_dir/fv3gfs_ocean_merge.sh
-               fi
-       fi
+  if [ $gtype = uniform ]; then
+    if declare -p ocn &>/dev/null;then
+      $script_dir/fv3gfs_ocean_merge.sh
+      err=$?
+      if [ $err != 0 ]; then
+        exit $err
+      fi
+    fi
+  fi

   set +x
```

You can find the updated script on Orion - /work/noaa/da/ggayno/save/ufs_utils.git/UFS_UTILS.sanath/ush

Reply:
> > Yes they are pregeneated and so "fixed". They can be regenerated and I have included a README in that directrory. If it needs to placed in an official location how do i Go about it ? I will clean it up as it does have some unnecessary files too.
> 
> Create a clean directory (no extraneous files) and we will ask @KateFriedman-NOAA for assistance.

We can open an issue here: https://github.com/NOAA-EMC/global-workflow/issues/new/choose

Reply:
Will make a clean  directory, test it out and then will start the process